### ISSUE TITLE
Feature Request: Send messages to ask users with a specific tag

### ISSUE THREAD

**Issue by leisenstein** [Nuordo/tags-for-slack#5]
I'd like to be able to send a message to all users with a specific tags by adding the tag to the message.

Ex: 
We have some unexpected outages in out QA servers. #qa #database 

This message works fo it to all users with the "qa" tag & "database" tag.  Make sure it's not case sensitive.

This would make Tags REALLY useful.

**Comment by mmarkus13** [comment]
Indeed; I was also looking for an application for slack where we could notify based on custom tags - instead notifying all by @here or @channel it would be super helpful to have custom tag notifications like @@dev @@python... etc.
